regression/cbmc-library/float-nan-check/test.desc

```diff
@@ -0,0 +1,24 @@
+CORE
+main.c
+--nan-check
+\[main.NaN.1\] line \d+ NaN on / in 0.0 / 0.0: FAILURE
+\[main.NaN.2\] line \d+ NaN on / in \(float\)n / return_value___builtin_inff: FAILURE
+\[main.NaN.3\] line \d+ NaN on \* in return_value___builtin_inff\$0 \* \(float\)0: FAILURE
+\[main.NaN.4\] line \d+ NaN on \+ in -return_value___builtin_inff\$1 \+ return_value___builtin_inff\$2: FAILURE
+\[main.NaN.5\] line \d+ NaN on \+ in return_value___builtin_inff\$3 \+ -return_value___builtin_inff\$4: FAILURE
+\[main.NaN.6\] line \d+ NaN on - in return_value___builtin_inff\$5 - return_value___builtin_inff\$6: FAILURE
+\[main.NaN.7\] line \d+ NaN on - in -return_value___builtin_inff\$7 - -return_value___builtin_inff\$8: FAILURE
+\[main.NaN.8\] line \d+ NaN on \+ in byte_extract_little_endian\(c, 0l, float\) \+ byte_extract_little_endian\(c, 0l, float\): SUCCESS
+\[main.NaN.9\] line \d+ NaN on / in \(double\)byte_extract_little_endian\(c, 0l, float\) / 0.0: SUCCESS
+\[main.NaN.10\] line \d+ NaN on \* in return_value___builtin_nanf \* return_value___builtin_nanf\$0: SUCCESS
+\[main.NaN.11\] line \d+ NaN on \+ in return_value___builtin_nanf\$1 \+ return_value___builtin_nanf\$2: SUCCESS
+\[main.NaN.12\] line \d+ NaN on - in return_value___builtin_nanf\$3 - return_value___builtin_nanf\$4: SUCCESS
+\[main.NaN.13\] line \d+ NaN on / in return_value___builtin_nanf\$5 / return_value___builtin_nanf\$6: SUCCESS
+\[main.NaN.14\] line \d+ NaN on \+ in return_value___builtin_nanf\$7 \+ \(float\)n: SUCCESS
+\[main.NaN.15\] line \d+ NaN on - in return_value___builtin_nanf\$8 - \(float\)n: SUCCESS
+\[main.NaN.16\] line \d+ NaN on \* in return_value___builtin_nanf\$9 \* \(float\)n: SUCCESS
+\[main.NaN.17\] line \d+ NaN on / in return_value___builtin_nanf\$10 / \(float\)n: SUCCESS
+^EXIT=10$
```

Comment:
Can you add a negative test for `main.NaN.18` so that adding additional checks will break this test.

Reply:
OK